env.example

```diff
@@ -329,3 +329,9 @@ JIBRI_LOGS_DIR=/config/logs
 # Container restart policy
 # Defaults to unless-stopped
 RESTART_POLICY=unless-stopped
+
+#OCTO
+ENABLE_OCTO=0
+
+#Octo BRIDGE_SELECTION_STRATEGY
+TESTING_OCTO_PROBABILITY=SplitBridgeSelectionStrategy
```

Comment:
@goacid Please change this env to `OCTO_BRIDGE_SELECTION_STRATEGY`. `TESTING_OCTO_PROBABILITY` is misleading here because for `service:web` it's value is `1` or `0`.

Reply:
That is my point, using the same var name, and if the var is set then is means activate it with the value inside.
No need to have activation and value in to separate var